src/elements/Label/Label.js

```diff
@@ -35,31 +34,34 @@ function Label(props) {
     useKeyOnly(circular, 'circular'),
     useKeyOnly(floating, 'floating'),
     useKeyOnly(horizontal, 'horizontal'),
+    useKeyOnly(image, 'image'),
     useKeyOnly(empty, 'empty'),
     useKeyOnly(tag, 'tag'),
     useValueAndKey(attached, 'attached'),
     useKeyOrValueAndKey(corner, 'corner'),
     useKeyOrValueAndKey(pointing, 'pointing'),
+    /* Don't change order of `above` and `below`, they must be after `pointing` */
+    useKeyOnly(above, 'above'),
+    useKeyOnly(below, 'below'),
```

Comment:
I think these should be `_meta.props.pointing` values.  Since they are not valid on their own:

```jsx
pointing
pointing='left'
pointing='right'
pointing='above'
pointing='below'
```

Then, we'll just have to write handling/tests to ensure the order is correctly implemented.  We won't be able to use common tests or lib use* methods in this case.